lib/ansible/modules/packaging/language/pip.py

```diff
@@ -235,6 +237,61 @@
                              'pip': 'import pkg_resources; print(pkg_resources.get_distribution("pip").version)'}
 
 
+class Package(Requirement):
+    def __init__(self, name_string, version_string=None):
+        if version_string is not None:
+            if version_string[0].isdigit():
+                name_string = "%s ==%s" % (name_string, version_string)
+            else:
+                name_string = "%s %s" % (name_string, version_string)
+        super(Package, self).__init__(name_string)
+
+    def have_version_specifier(self):
+        if len(self.specs) == 0:
+            return False
+        return True
+
+    def get_name(self):
+        return self.name.lower()
+
+    def satisfied_by(self, version_to_test):
+        return self.specifier.contains(version_to_test)
+
+    def install_command(self):
```

Comment:
1. `@property`.
2. It's not an installation command, it's a version spec.
3. You don't need it, as it's stringified automatically during string interpolation, under 3.6.5:
```python
In [3]: from pkg_resources import Requirement

In [5]: rqc = Requirement('cherrypy')

In [11]: '%s ' % rqc
Out[11]: 'cherrypy '

```

2.6.9:
```python
$ python
Python 2.6.9 (unknown, Apr 10 2018, 17:32:50)
[GCC 7.3.0] on linux4
Type "help", "copyright", "credits" or "license" for more information.
>>> from pkg_resources import Requirement
/home/wk/.pyenv/versions/2.6.9/lib/python2.6/site-packages/pkg_resources.py:17: DeprecationWarning: the sets module is deprecated
  from sets import ImmutableSet
>>> rqc = Requirement('cherrypy')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() takes exactly 4 arguments (2 given)
>>> rqc = Requirement.parse('cherrypy')
>>> '%s ' % rqc
'cherrypy '
>>>
```